core/common/src/main/java/alluxio/util/UnderFileSystemUtils.java

```diff
@@ -158,5 +160,14 @@ public static boolean isSwift(UnderFileSystem ufs) {
     return "swift".equals(ufs.getUnderFSType());
   }
 
+  /**
+   * @param uri the UFS path
+   * @return the bucket or container name of the object storage
+   */
+  public static String getBucketName(AlluxioURI uri) {
+    Preconditions.checkState(isObjectStorage(uri.toString()));
+    return uri.getAuthority();
```

Comment:
Right, I understand why `getAuthority` is correct compared to `getHost`. I was just commenting on the fact that we are using the bucket as the authority when it looks more like the first component of the path. However, since we don't use something like `s3:///`, it happens to work out for us.